src/coreclr/jit/emitloongarch64.cpp

```diff
@@ -4918,7 +4918,7 @@ regNumber emitter::emitInsTernary(instruction ins, emitAttr attr, GenTree* dst,
 
         if (needCheckOv)
         {
-            emitIns_R_R_R(INS_or, attr, REG_R21, nonIntReg->GetRegNum(), REG_R0);
+            emitIns_R_R_I(INS_ori, attr, REG_R21, nonIntReg->GetRegNum(), 0);
```

Comment:
Is this change necessary? Does it have any difference between `OR zr` and `ORI imm#0`?
Which one is used consistently in codegen for LA?

Reply:
Here `OR zr` and `ORI imm#0` do the same thing, but `ORI imm#0` is used consistently for `$rd=$rn`.